src/web/content/static/css/letterpress.css

```diff
@@ -53,6 +53,7 @@
     --medium-dark-gray: #bcbcbc;
     --light-gray:#CECECE;
     --medium-light-gray: #D9D9D9;
+    --underline-border-bottom: 0.125rem solid #DBA0C7;
```

Comment:
Using this variable to remove repetition. See the 4 items below that got replaced with this.